app/MMB/src/scenes/passenger/PassengerDetailContainer.js

```diff
@@ -11,7 +11,7 @@ type Props = {||};
 
 export default class PassengerDetailContainer extends React.Component<Props> {
   renderInner = (renderProps: PassengerDetailContainerQuery) => {
-    return <PassengerDetail booking={renderProps.booking} />;
+    return <PassengerDetail booking={renderProps.node} />;
```

Comment:
Was the query already updated before?

Reply:
Yes, this was a mistake I just fixed it... 🙂